pkg/volume/util/hostutil/hostutil_windows.go

```diff
@@ -91,25 +88,17 @@ func (hu *HostUtil) MakeRShared(path string) error {
 	return nil
 }
 
-func isSystemCannotAccessErr(err error) bool {
-	if fserr, ok := err.(*fs.PathError); ok {
-		errno, ok := fserr.Err.(syscall.Errno)
-		return ok && errno == windows.ERROR_CANT_ACCESS_FILE
-	}
-
-	return false
-}
-
 // GetFileType checks for sockets/block/character devices
-func (hu *(HostUtil)) GetFileType(pathname string) (FileType, error) {
+func (hu *HostUtil) GetFileType(pathname string) (FileType, error) {
 	filetype, err := getFileType(pathname)
+	if err == nil {
+		return filetype, nil
+	}
 
-	// os.Stat will return a 1920 error (windows.ERROR_CANT_ACCESS_FILE) if we use it on a Unix Socket
+	// os.Stat will return an unknown error (based on different Windows versions) if we use it on a Unix Socket
 	// on Windows. In this case, we need to use a different method to check if it's a Unix Socket.
-	if isSystemCannotAccessErr(err) {
-		if isSocket, errSocket := filesystem.IsUnixDomainSocket(pathname); errSocket == nil && isSocket {
-			return FileTypeSocket, nil
-		}
+	if isSocket, errSocket := filesystem.IsUnixDomainSocket(pathname); errSocket == nil && isSocket {
```

Comment:
As long as `getFileType` returns an error, `IsUnixDomainSocket` must be used to determine whether it is a sock file.